release-lock-file-spec.md

```diff
@@ -23,52 +152,69 @@ single JSON object.
 
 ## Filename
 
-When creating a release, convention is to name this document `<version>.json`.
+When creating a release, convention is to name this document using one of the following conventions.
 
-* The `1.0.0` Release would be named `1.0.0.json`
+* In cases where multiple *Release Lockfiles* are being store alongside one another, convention is to use `<version>.json`.  For example, the `1.0.0` release would be named `1.0.0.json`.  
+* In cases where a single *Release Lockfile* is being stored, convention is to use `lock.json`.
 
-Package managers may keep their own copy of this file as part of their package installation process.  In these cases it is up to the package manager to decide on a naming convention.  Package managers **should** keep the `.json` file extension.
+Package managers **should not** rely on these naming conventions nor are they
+required to follow them.
```

Comment:
Not to be a curmudgeon, but if package managers shouldn't rely on them, should this section exist in the spec? On my end, I'm actually calling the files `lock.json` for consistency of implementation. This is a package manager implementation decision, so I think we should remove it from the sepc.